src/server/node_manager.c

```diff
@@ -1459,6 +1459,14 @@ set_vnode_state(struct pbsnode *pnode, unsigned long state_bits, enum vnode_stat
 	}
 
 	if (pnode->nd_state & INUSE_PROV) {
+#if MOM_ALPS
```

Comment:
Why is this conditional compilation block needed? Is the code inside the block not generic and will it fail on a non-cray machine?

Reply:
Removed this macro in previous commit.
No it will not fail on non-cray machine.